docs/modules/usage/troubleshooting/troubleshooting.md

```diff
@@ -83,18 +86,20 @@ AttributeError: 'NoneType' object has no attribute 'request'
 
 [GitHub Issues](https://github.com/OpenDevin/OpenDevin/issues?q=is%3Aissue+is%3Aopen+404)
 
-This usually happens with local LLM setups, when OpenDevin can't connect to the LLM server.
+This usually happens with *local* LLM setups, when OpenDevin can't connect to the LLM server.
 See our guide for [local LLMs](llms/localLLMs) for more information.
 
 ### Workarounds
 
-- Check your `LLM_BASE_URL`
-- Check that ollama is running OK
-- Make sure you're using `--add-host host.docker.internal:host-gateway` when running in docker
+* Check your `LLM_BASE_URL` in your config.toml (if it exists) or command line call
```

Comment:
not accurate; it's `base_url` under an LLM config group in config.toml

Reply:
Oh, thanks for that, fixed here.

But wanted to ask for your feedback about another entry I'd like to add, but I'm not quite yet sure if it is actually 2 separate entries that touch sessions (local storage, jwt secret) and how to word it best:

## Sessions are not restored

### Symptoms

OpenDevin usually asks whether to resume or start a new session when opening the UI.
But clicking "Resume" still starts a fresh new chat.

### Details

With a standard installation as of today session data is stored in memory.
Currently, if OpenDevin's service is restarted, previous sessions become
invalid (a new secret is generated) and thus not recoverable.

### Workarounds

* Change configuration to make sessions persistent by editing the `config.toml`
file (in OpenDevin's root folder) by specifying a `file_store` and an
absolute `file_store_path`:

```toml
file_store="local"
file_store_path="/absolute/path/to/opendevin/cache/directory"
```